data/map.txt

```diff
@@ -30317,9 +30317,9 @@ planet Millrace
 	landscape land/mfield1
 	description `Millrace is an overpopulated world, full of factories and manufacturing plants. It is one of several planets in this region of space where people trying to escape from the crowded squalor and polluted cities of Earth come looking for work. Most of the goods manufactured here end up being used by the Syndicate for constructing starships or driving industry on other worlds.`
 	description `	The Syndicate is a fair employer: living conditions here may be squalid, but most workers are earning a living wage and some are able to save up for a new life on a more pleasant planet. The Syndicate trusts there will never be a shortage of new immigrants to replace them.`
-	spaceport `This spaceport is ugly.`
-	spaceport `	As the population of Millrace has grown, the spaceport has been expanded to support the increase in immigration; you can recognize the different sections because in the newer ones, the bare concrete walls are lighter and have accumulated less soot and grime.`
-	spaceport `	Most of the restaurants are of the greasy spoon variety.`
+	spaceport `This spaceport is ugly. It was designed not for the pleasure of sight, but for processing as many people through it as possible, without collapsing onto itself.`
+	spaceport `	As the population of Millrace has grown, the spaceport has been continually expanded to support the increase in immigration; you can recognize the different sections, because in the newer ones the bare concrete walls are lighter and have accumulated less soot and grime. Most of the restaurants are of the greasy spoon variety.`
```

Comment:
```suggestion
	spaceport `	As the population of Millrace has grown, the spaceport has been continually expanded to support the increase in immigration; you can recognize the different sections because in the newer ones the bare concrete walls are lighter and have accumulated less soot and grime. Most of the restaurants are of the greasy spoon variety.`
```